authnid/make_db.py

```diff
@@ -0,0 +1,29 @@
+import psycopg2
+import psycopg2.extras
+
+def connect_db(uri):
+    conn = psycopg2.connect(uri, cursor_factory=psycopg2.extras.DictCursor)
+    conn.set_session(autocommit=True)
+    return conn
+
+def make_cursor(connection):
+    return connection.cursor()
+
+def make_db(config):
+    connection = connect_db(config['DATABASE_URI'])
+    cursor = make_cursor(connection)
+    cursor.execute('''
+        CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+        CREATE TABLE IF NOT EXISTS users (
```

Comment:
I get that we're still in rough mode, but I strongly feel that implementing DB migrations now will make things much smoother going forward. I strongly recommend going that route now versus starting with hard-coded schemas.